### PR TITLE
Tailwindsetup

### PR DESCRIPTION
**Pull request by ArthiRagothaman**
Added icons and created Storybook(Navigationbar component)

### CODE REVIEW

**Review by ReallyBoard123 — COMMENTED** [review]
Hey, I have added more comments on the files. Please take a look at them :)

Tailwind config file will contain all the information that you will consistently use throughout your components.

File: src/stories/Navigationbar.stories.tsx [hunk 1]
```diff
@@ -0,0 +1,18 @@
+import Navigationbar from "../Layout/Navigationbar";
```
[the remark this answers is not in the document]
Comment (n1lsw) [reply src/stories/Navigationbar.stories.tsx:1]: We created a new branch from tailwind setup branch called navbar
We deleted everything Navbar related on the tailwind setup branch

File: .storybook/preview.js [hunk 2]
```diff
@@ -16,5 +18,5 @@ export const parameters = {
     },
   },
 };
-
+withRouter;
```
Comment (NoHara42) [inline .storybook/preview.js:21]: what is this about?

Comment (NoHara42) [reply .storybook/preview.js:21]: Why is this included in the tailwindsetup issue PR? Shouldn't this have been a part of the Routing issue PR?

File: package.json [hunk 3]
```diff
@@ -29,7 +29,7 @@
     "@storybook/addon-interactions": "^7.0.0-beta.19",
     "@storybook/addon-links": "^7.0.0-beta.19",
     "@storybook/blocks": "^7.0.0-beta.19",
-    "@storybook/react": "^7.0.0-beta.19",
+    "@storybook/react": "7.0.0-beta.19",
```
Comment (NoHara42) [inline package.json:32]: What's the purpose of this?

**Review by NoHara42 — CHANGES_REQUESTED** [review]
Please see comments

File: .storybook/preview.js (hunk 2 above)
Comment (TheDoplarEffect) [reply .storybook/preview.js:21]: Yes, we tried to strip out references to other components but this must have slipped through.  It should only be the tailwind config file as far as I understand

**Comment by TheDoplarEffect** [comment]
We have copy pasted the other teams config and index.css file.  Should be good to finally merge into main

**Review by ReallyBoard123 — APPROVED** [review]

File: .storybook/preview.js [hunk 4]
```diff
@@ -1,3 +1,5 @@
+import { withRouter } from "storybook-addon-react-router-v6";
```
Comment (NoHara42) [inline .storybook/preview.js:1]: Don't commit unused imports

**Review by NoHara42 — APPROVED** [review]

**Pull request merged**